### PR TITLE
Many broken internal link fixes.

### PR DESCRIPTION
**Pull request by dannon**
Some targeted, some global, addressing stuff identified in #860 

Also converted a bunch of the older `/src/<whatever>/index.md` paths to just `/<whatever>/` in the process.

Local links status after this:

### 📝Link summary of 1571 pages checked

| 🔍 Checked   | ✅ Successful | 🚫 Errors   |
| ---: | ---: | ---: |
| 49723 | 49550 | 173

### CODE REVIEW

**Comment by dannon** [comment]
(the global ones are suspect; I'm reviewing and iterating while this is in draft -- I hope we can fix many if not most of these links with some regex-fu)

File: content/events/2021-01-sv-with-ont/index.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ tease: "A GalaxyPro Webinar"
 continent: GL
 location: "GalaxyWorks, Online, Global"
 location_url: "https://galaxyworks.io/webinars/"
-external_url: "/events/2021-01-sv-wtih-ont/sv-with-ont.pdf"
+external_url: "/events/2021-01-sv-with-ont/sv-with-ont.pdf"
```
Comment (NickSto) [inline content/events/2021-01-sv-with-ont/index.md:9]: Is it worth leaving a redirect to the original url? Not sure if anyone's hotlinked to it.

**Review by NickSto — APPROVED** [review]
I built and tested it locally, and did spot checks on each commit. Looks like it all works to me. Three thumbs up!

File: content/events/2021-01-sv-with-ont/index.md (hunk 1 above)
Comment (dannon) [reply content/events/2021-01-sv-with-ont/index.md:9]: Probably not for this one

File: src/build/run.mjs [hunk 2]
```diff
@@ -56,12 +56,17 @@ function main(rawArgv) {
...
-            cwd: "./content",
-            overwrite: false,
-            parents: true,
-        });
+        // Copy static images for direct reference to dist -- only when doing a
+        // full build.  We hook into the exit this way to let Gridsome do its
+        // thing first.
+        if (command === "build") {
+            console.log("Coping integrated static content (png, jpg, pdf, gif) to dist.");
+            cpy(["**/*.png", "**/*.jpg", "**/*.pdf", "**/*.gif"], "../dist", {
```
[the remark this answers is not in the document]
Comment (dannon) [reply src/build/run.mjs:64]: Yeah, let me add that and look to make sure there weren't any others in the list.

**Comment by dannon** [comment]
@NickSto I'll merge upstream back in here and add svg and check for other artifacts we'd want to directly link.

**Comment by dannon** [comment]
@NickSto Thanks for checking through it -- it should be good to go now.

**Pull request merged**